cupy/_math/misc.py

```diff
@@ -399,7 +399,24 @@ def nan_to_num(x, copy=True, nan=0.0, posinf=None, neginf=None):
     return _nan_to_num(x, nan, posinf, neginf, out=out)
 
 
-# TODO(okuta): Implement real_if_close
+def real_if_close(x, tol=100):
+    """If input is complex with all imaginary parts close to zero, return real
+    parts.
+    “Close to zero” is defined as tol * (machine epsilon of the type for x).
+
+    .. seealso:: :func:`numpy.real_if_close`
+
+    """
+    x = cupy.asanyarray(x)
```

Comment:
Most CuPy's functions don't allow nested sequence or numpy.ndarray to avoid implicit host-to-deivce data transfer.

Reply:
I am a little confused as to what is to be done here.😅

Reply:
```suggestion
```
It's OK to just delete this line!

Reply:
I did do that initially, but it will still allow ndarrays, isn't it?